Doc/library/sqlite3.rst

```diff
@@ -1255,39 +1264,33 @@ but as a Unix timestamp.
 Converting SQLite values to custom Python types
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Writing an adapter lets you send custom Python types to SQLite. But to make it
-really useful we need to make the Python to SQLite to Python roundtrip work.
+To be able to convert SQLite value to custom Python types, we use _converters_.
 
-Enter converters.
-
-Let's go back to the :class:`Point` class. We stored the x and y coordinates
-separated via semicolons as strings in SQLite.
-
-First, we'll define a converter function that accepts the string as a parameter
-and constructs a :class:`Point` object from it.
+Let's revisit the ``Point`` class example from above;
+the coordinate pair is stored in the database as a semicolon separated string.
+We define a converter that accept a string, and return a ``Point`` object.
```

Comment:
Not sure about the value of this change; the existing text seems fine to me.

Also, the ``Point`` class seems to be coming up again and again, but this PR deletes it from lines 1226-1228 -- is that wise?

Reply:
> Not sure about the value of this change; the existing text seems fine to me.

Ok, but I don't like "enter converters".

> Also, the Point class seems to be coming up again and again, but this PR deletes it from lines 1226-1228 -- is that wise?

Hm, yeah, I'll reconsider it. Thanks!